R/roles.R

```diff
@@ -311,15 +305,15 @@ update_role <- function(recipe, ..., new_role = "predictor", old_role = NULL) {
 #' @export
 remove_role <- function(recipe, ..., old_role) {
   if (rlang::is_missing(old_role)) {
-    rlang::abort('argument "old_role" is missing, with no default.')
+    cli::cli_abort("argument {.val old_role} is missing, with no default.")
```

Comment:
```suggestion
    cli::cli_abort("argument {.arg old_role} is missing, with no default.")
```

This will cause a snapshot change, unlike the change you had chosen already, but it feels worth using cli's syntax faithfully since its output may change for a given inline markup anyway.